packages/api/lib/lzards.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
```

Comment:
Is there a reason this is in the API package versus it's own package and/or not typescripted?

It doesn't look like it's actually using anything internally in the API code.  

We should consider pulling this out into it's own module, but new modules *definitely* should be typescripted in packages that support it.

Reply:
That's a great question. The short answer is, "Not anymore". Since removing the api endpoints that were calling /lib/lzards.js there is no longer a need for that code to be in the api package. I can create a typescript package and move it over.

Reply:
👍🏻   I don't want to 11th hour scope creep this, depending on where we are with tickets/etc, but if you're willing it will likely make maintenance better long term.

Reply:
completed